metricbeat/module/php_fpm/proc/_meta/fields.yml

```diff
@@ -0,0 +1,74 @@
+- name: proc
+  type: group
+  description: >
+    `proc` contains the metrics that were obtained from a PHP process managed by
+    the PHP-FPM process pool.
+  fields:
+    - name: pid
+      type: keyword
+      description: >
+        The PID of the process.
+
+    - name: state
+      type: keyword
+      description: >
+        The state of the process (`Idle`, `Running`, ...).
+
+    - name: start_time
+      type: long
+      description: >
+        The date and time the process has started.
+
+    - name: start_since
+      type: long
+      description: >
+        Number of seconds since the process has started.
+
+    - name: requests
+      type: long
+      description: >
+        The number of requests the process has served.
+
+    - name: request_duration
```

Comment:
request.duration.us if it is micro seconds. See https://www.elastic.co/guide/en/beats/libbeat/5.1/event-conventions.html